solr_conf/conf/solrconfig.xml

```diff
@@ -158,16 +169,27 @@
     </lst>
   </requestHandler>
 
+
   <requestDispatcher handleSelect="true" >
     <requestParsers enableRemoteStreaming="true" multipartUploadLimitInKB="2048" />
   </requestDispatcher>
 
   <requestHandler name="/analysis/field" startup="lazy" class="solr.FieldAnalysisRequestHandler" />
   <requestHandler name="/update" class="solr.UpdateRequestHandler"  />
+
   <updateHandler class="solr.DirectUpdateHandler2">
     <updateLog>
       <str name="dir">${solr.data.dir:}</str>
     </updateLog>
+    <autoCommit>
```

Comment:
It looks like these particular values were chosen to work well with
ActiveFedora
projecthydra/active_fedora:c9733e3015710cbbc6c73fa4a85b5c59f35fd773.  Based
on this I'm comfortable with merging this change.